src/submission/NodeSubmissionClient.ts

```diff
@@ -16,44 +16,35 @@ export class NodeSubmissionClient extends DefaultSubmissionClient {
     this.configurationVersionHeader = this.configurationVersionHeader.toLowerCase();
   }
 
-  public sendRequest(config:Configuration, method:string, path:string, data:string, callback: (status:number, message:string, data?:string, headers?:Object) => void): void {
-    function complete(response:http.IncomingMessage, responseBody:string, responseHeaders:Object) {
-      var message:string;
-      if (response.statusCode === 0) {
-        message = 'Unable to connect to server.';
-      } else if (response.statusCode < 200 || response.statusCode > 299) {
-        message = response.statusMessage || (<any>response).message;
-      }
+  public sendRequest(config: Configuration, method: string, path: string, data: string, callback: NodeSubmissionCallback): void {
```

Comment:
The callback signature (NodeSubmissionCallback) is just a shortcut for what has been there before. In TypeScript, this still works as long as signatures are compatible.